src/library.js

```diff
@@ -3267,6 +3267,40 @@ LibraryManager.library = {
   },
   emscripten_asm_const_double: 'emscripten_asm_const_int',
 
+  $JsValStore: {
```

Comment:
I am not sure if we should regard this a global Emscripten wide machinery, at least at this point. Maybe this would be best to locate in library_webgpu.js?

Reply:
Moved to library_html5_webgpu.js and html5_webgpu.h